yarn-project/sequencer-client/src/sequencer/sequencer.ts

```diff
@@ -261,6 +261,8 @@ export class Sequencer {
       await this.publishL2Block(block, aggregationObject, proof);
       this.log.info(`Submitted rollup block ${block.number} with ${processedTxs.length} transactions`);
     } catch (err) {
+      // wipe the mempool
+      await this.p2pClient.deleteTxs((await this.p2pClient.getTxs()).map(tx => tx.getTxHash()));
```

Comment:
Shouldn't we just wipe out the txs we picked up, and not the entire mempool?

Reply:
Done ✅